_data/toc.yaml

```diff
@@ -591,10 +591,16 @@ reference:
         title: docker scout compare
       - path: /engine/reference/commandline/scout_cves/
         title: docker scout cves
+      - path: /engine/reference/commandline/scout_entitlement/
+        title: docker scout entitlement
```

Comment:
I noticed that the new commands don't have the `experimental` option set; are they considered "stable"?

TBH, I was a bit confused by this command; this feels like we should revisit having a `docker hub` CLI plugin (to manage configuration on Docker Hub); should we revisit https://github.com/docker/hub-tool, and make it into an actual `docker hub` CLI plugin?

Reply:
that would be cool!

Reply:
Yeah, it felt a bit out of scope for scout, as it's more "management" functions for Docker Hub (and there's many other options / features to be set, not just `scout`). In `scout`'s case, this is a feature that depends on a hosted solution that's Docker Hub specific (so maybe a "bit" of a grey area), but it feels a bit like crossing boundaries / responsibilities, and cluttering the UX.